app/controllers/proposal_controller.rb

```diff
@@ -89,9 +90,13 @@ def destroy
       return
     end
 
-    @event.save(validate: false)
-    redirect_to conference_program_proposal_index_path(conference_id: @conference.short_title),
-                notice: 'Proposal was successfully withdrawn.'
+    if @event.save
```

Comment:
> Do we really need to validate when we withdraw the event? 

Not really. I had asked about proposed change [here](https://github.com/openSUSE/osem/pull/867#discussion_r55916850). My reasoning is that:
* if are to use `@event.save(validate: false)`, we should do the same in other similar actions (`confirm` and `restart`). @hennevogel said that I should go with `if..else`
* We should do `if @event.save` cause save can fail for reasons other than validation, for ex: our database connection is lost, running out of memory, some obscure IO/socket error (source: [this blog](https://robots.thoughtbot.com/save-bang-your-head-active-record-will-drive-you-mad))